### ISSUE TITLE
Bug in table when column is invisible

### ISSUE THREAD

**Issue by Rodbjartson** [olifolkerd/tabulator#1299]
Setting a row's visibility seems to be an issue in tabulator, I am however not able to pinpoint why and when this happens.

Look at these screenshots:
![thirdshot](https://user-images.githubusercontent.com/11444201/43189644-54ffdafa-8ff7-11e8-93ff-f3c05d150b6c.png)
![secondshot](https://user-images.githubusercontent.com/11444201/43189642-51f91a74-8ff7-11e8-9d67-f1540c74a123.png)
![firstshot](https://user-images.githubusercontent.com/11444201/43189632-4a5d832c-8ff7-11e8-967a-9e9bf654e64f.png)
![fourthshot](https://user-images.githubusercontent.com/11444201/43189650-58340dcc-8ff7-11e8-8dee-fd712275ad9f.png)

The first picture shows my columns, notice the column with field: "ID_WOITEM_SEQ"
It is lined up as last in the first column group, and the second picture shows the table as it should be. The column is hidden.

The third picture shows how I moved the column to the last column group, everything is still the same.
However, as the fourth picture shows, it causes tabulator to show a weird looking table, almost as it has not been drawn correctly. 

Is this a bug, or am I missing something?

**Comment by Rodbjartson** [comment]
**_The whole tabulator constructor:_**

```
$("#item-table-modal-confirmedOrder").tabulator({
                // set height of table, this enables the Virtual DOM and improves render speed dramatically (can be any valid css height value)
                // layout: "fitData", //fit columns to width of table (optional)
                height: 340,
                fitColumns : true,
                selectable: true,     //true means we can select multiple rows   
                
                
                columns: [ //Define Table Columns
                    {//create column group
                        title: "Vareinfo",
                        columns: [
                            { title: "Varenr", field: "ID_ITEM", align: "center" },
                            { title: "Varenavn", field: "ITEM_DESC", align: "center" },
                            { title: "Varegruppe", field: "ITEM_CATG_DESC", align: "center" },
                            { title: "VaregruppeID", field: "ID_ITEM_CATG", visible: false }, //create hidden column. only so that we can easily provide the id_item_catg
                            { title: "OrdreNrVare", field: "ID_WOITEM_SEQ", visible: false },
                        ],
                    },
                    
                    //{ title: "Totalkostnad", field: "TOTALCOST", align: "center", formatterParams: { precision: 2 } },
                    
                    {//create column group
                        title: "Prisdetaljer",
                        columns: [
                            { title: "Rab", field: "DISCOUNT", align: "center" },
                            { title: "Salgspris", field: "ITEMPRICE", align: "center" },
                            { title: "Kostpris", field: "BUYCOST", align: "center" },
                            { title: "Nettopris", field: "NETPRICE", align: "center" },
                            { title: "Veil. pris", field: "INDIACTIVEPRICE", align: "center" },
                            
                        ],
                    },
                    {//create column group
                        title: "Leveringsdetaljer",
                        columns: [
                            
                            
                            { title: "Ant. bestilt", field: "ORDERQTY", align: "center" },
                            { title: "Ant. levert", field: "DELIVERED_QTY", align: "center" },
                            { title: "Ankommet", field: "REMAINING_QTY", align: "center", editor: "number", editorParams: { min: 0, max: 999999, step: 1 }, cellEdited: function (cell) {
                            //cell - cell component
                                verifyInputCellEdit(cell);
                            }
                            },
                            { title: "Resting", field: "REST_FLG", align: "center", formatter: "tickCross" },
                            { title: "Ordre", field: "ID_WO_NO_AND_PREFIX", align: "center" },
                            

                        ],
                    },
                    
                    

                    
                ],

                ajaxResponse: function (url, params, response) {
                    console.log("url is: " + url);
                    console.log("params is: " + params);
                    console.log(typeof (response.d[0].TOTALCOST));
                    //url - the URL of the request
                    //params - the parameters passed with the request
                    //response - the JSON object returned in the body of the response.

                    return response.d; //return the d property of a response json object
                },
                dataLoading: function (data) { //we need this because data that comes in is strings, cant be 
                    //data - the data loading into the table
                },

                cellEditing: function (cell) {
                    //cell - cell component
                    
                    //cell = cell.getRow().deselect();
                     
                },

                ajaxResponse: function (url, params, response) {
                   
                    //url - the URL of the request
                    //params - the parameters passed with the request
                    //response - the JSON object returned in the body of the response.

                    return response.d; //return the d property of a response json object
                },
                dataLoading: function (data) { //we need this because data that comes in is strings, cant be 
                    //data - the data loading into the table
                },




            });
```

**Comment by olifolkerd** [comment]
Hey @Rodbjartson 

Sorry for the delay in getting back to you,

That does look like strange behaviour, any chance you could setup a demo that replicates the issue in code pen so i can see it in action?

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @Rodbjartson 

There were some issue with the column grouping system that were resolved in version 4.0, give it a try with that and see if you are still having issues.

Cheers

Oli :)
